Eatery/Controllers/Eateries/Campus/CampusEateriesViewController.swift

```diff
@@ -162,6 +171,18 @@ class CampusEateriesViewController: EateriesViewController {
 
         navigationItem.searchController = searchController
     }
+    
+    private func setUpLocationManager() {
+        if CLLocationManager.locationServicesEnabled() {
```

Comment:
IMO move the location manager setup out of the lazy initializer and into here

Reply:
Should I keep it a lazy var (making it a private lazy var) or remove lazy from its declaration (making it just a private var)

Reply:
I think `private let`

Reply:
When I remove lazy I get the following error on locationManager.delegate = self

Cannot assign value of type '(CampusEateriesViewController) -> () -> CampusEateriesViewController' to type 'CLLocationManagerDelegate?'

Reply:
I think the location manager should be `private let`, because IMO `private let` declarations are cleaner: why have the set up split between two places when we could have it in just one?

So I think you should think a bit more about how to solve this issue. Let me know if you really cannot figure it out